apps/site/assets/css/_events.scss

```diff
@@ -40,3 +42,30 @@
 .event-inline-block {
   display: inline-block;
 }
+
+.m-event__date-circle {
+  display: flex;
+  flex-direction: column;
+  height: 4rem;
```

Comment:
We have a Sass variable called `$base-spacing` which if I recall correctly is equivalent to `1rem` -- can you adjust these styles to make use of this variable?